libraries/botbuilder-lg/src/expander.ts

```diff
@@ -266,7 +266,10 @@ export class Expander extends AbstractParseTreeVisitor<unknown[]> implements LGT
                 for (const refValue of templateRefValues.get(templateRefValueKey)) {
                     tempRes.push(
                         JSON.parse(
-                            JSON.stringify(res).replace(templateRefValueKey, refValue.toString().replace(/"/g, '\\"'))
+                            JSON.stringify(res).replace(
+                                templateRefValueKey,
+                                refValue.toString().replace(/"|\\/g, '\\"')
```

Comment:
We added a `\` because in the issue description it says that the backslashes in the input aren't being escaped but they shouldn't be replaced with \" I think.